pkg/account/downloader/internal/http/http_utils.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package http
+
+import (
+	"net/http"
+)
+
+func closeResponseBody(resp *http.Response) {
+	_ = resp.Body.Close()
+}
+
+func handleClientResponseError(resp *http.Response, clientErr error) error {
+	return clientErr
```

Comment:
Currently, this won't treat HTTP errors as an error. I don't think that's how the downloader should behave? I can't see any checks for 4xx/5xx status errors anywhere else in the code. There's more than the 404 exclusion to the TODO below unless I'm mistaken

Reply:
For now, I am going to remove this part.
As I `download` is always getting the whole configuration, 404 is also an error in processing if some requested resources are absent.

There will be additional PR in which will this case be present and reused in delete/deploy